src/fabric/src/fabric2_server.erl

```diff
@@ -254,14 +255,25 @@ find_cluster_file([{custom, undefined} | Rest]) ->
     find_cluster_file(Rest);
 
 find_cluster_file([{Type, Location} | Rest]) ->
+    Msg = #{
+        in => fdb_connection_setup,
```

Comment:
Is the omission of `what` key in this module intentional?

Reply:
No, good catch. Wondering if that `in` ought to be a `what`; there aren't any other messages with the same value of `in`

Reply:
I think the `status` could become `what` as well. Except the `status => ok` case.

Reply:
I could see that too. I have a slight preference for `what => fdb_connection_setup`